.github/CONTRIBUTING.md

```diff
@@ -37,15 +38,30 @@ PlaceholderAPI provides a feature to have expansions (separate jar files) for pl
 In those cases should you report the issue to the issue tracker of the expansion or plugin.
 
 ## Pull requests
-As an open source project are we welcoming all contributions to improve PlaceholderAPI, being it changes to its code like bug fixes or new features, or contributions to its documentation such as the [Wiki] or the Javadoc.
+As an open source project are we welcoming all contributions to improve PlaceholderAPI, being it changes to its code, or contributions to its documentation such as the [Wiki] or the Javadocs.
+
+**When contributing, make sure to both base of and target the mentioned branch. Pull requests targeting the wrong branch may get closed without a warning.**
 
 ### Code contributions
-Any contributions to PlaceholderAPI's code should be done towards the [`development`][dev] branch. Targeting the `master` branch in your Pull request may get it closed without warning.  
-Additionally should you follow the project's codestyle which you can find in the [`config/style`][style] directory.
+> **Source and Target Branch:** [`master`][master]
+
+When contributing towards the code of PlaceholderAPI, be it new features or just bug fixes, should your changes follow the general code styling used in the project.  
```

Comment:
Last part of the sentence seems off. Maybe "your changes should follow the ..."?